torch/csrc/jit/script/compiler.cpp

```diff
@@ -908,10 +908,6 @@ struct to_ir {
       throw ErrorReport(stmt) << "Iteration variable unpacking is not supported";
     }
 
-    if (targets[0].kind() != TK_VAR) {
```

Comment:
was this impossible to hit?

Reply:
Actually, on second thought I think we would be able to hit this from the string frontend (if the string frontend supported starred expressions). Let me add this back and make a new issue